traitlets_documenter/class_traitlets_documenter.py

```diff
@@ -0,0 +1,87 @@
+# Modified from enthought/trait-documenter/trait_documenter/class_trait_documenter.py  # noqa
+
+# LICENSE: OSI Certified Open Source Software
```

Comment:
kit not need to move the copyright from trait-documenter just mention in a readme.rst in the trailtets_documenter that this code is insipired from the trait_documenter package.